modules/atlas/src/test/resources/data/neurosciencegraph/atlas/imagestack/v0.1.0/236174_imagestack.json

```diff
@@ -0,0 +1,24 @@
+{
+  "@context": "{{base}}/contexts/neurosciencegraph/core/data/v0.1.0",
+  "@type": [
+    "nsg:ParcellationImageData",
+    "nsg:ImageStack"
+  ],
+  "name": "Image stack for whole brain mouse id 236174",
+  "sliceDirection": "Coronal",
+  "sliceWidth": 22000,
+  "sliceHeight": 38000,
+  "numberOfSlices": 12058,
+  "sliceResolution": {
+    "widthResolution": 0.3,
+    "heightResolution": 0.3,
+    "unitText": "micron"
+  },
+  "sliceInterval": {
+    "sliceIntervalValue": 1.0,
+    "unitText": "micron"
+  },
+  "distribution": {
+    "accessURL": "/gpfs/bbp.cscs.ch/project/proj39/data/whole_brain_mouse/236174/"
```

Comment:
accessURL should be a URI (an http based one). Is whole_brain_mouse/236174 available somewhere ? Otherwise use http://projectpath/whole_brain_mouse/236174 for an example.